RecoLocalCalo/Configuration/python/hcalLocalReco_cff.py

```diff
@@ -79,6 +86,11 @@
         produceSoA = False
     )
 )
+#--- HCAL-only workflow for Run 2 on GPU
+from RecoLocalCalo.HcalRecProducers.hcalRecHitSoAToLegacy_cfi import  hcalRecHitSoAToLegacy 
+(alpaka & ~run3_HB).toModify(hbheprereco,
+    cuda = hcalRecHitSoAToLegacy.clone()
```

Comment:
Shouldn't this be `cpu = ...` ?

Reply:
changed to `cpu` in 97d3bc4114273cc29e8f9ad94713d3adc10c7a5a